.github/workflows/ci.yml

```
@@ -58,11 +58,20 @@ jobs:
       - name: rustdoc
         run: RUSTDOCFLAGS="-Dwarnings" ./miri doc --document-private-items
 
+  coverage:
+    name: Coverage report
```

```suggestion
    name: coverage report
```
Just to match the style of the other jobs